src/fetching/putEventResponse.ts

```diff
@@ -0,0 +1,22 @@
+import apiUrl from '../utils/apiUrl';
+
+import { ZetkinMembership } from '../types/zetkin';
+
+export default function putEventResponse(event : number, org : number | undefined) : Promise<ZetkinMembership> {
```

Comment:
What happens if `org` is `undefined` here? Will the code work? I don't think so, and hence `undefined` should not be an acceptable value for `org`.

Luckily enough, it's very unlikely that it will ever be undefined, because this function should only be used in cases where an event, and hence the org it belongs to, is known.

Reply:
See my reply above.